lib/node_modules/@stdlib/stats/base/dmskrange/README.md

```diff
@@ -184,6 +176,123 @@ console.log( v );
 
 <!-- /.examples -->
 
+<!-- C usage documentation. -->
+
+<section class="usage">
+
+### Usage
+
+```c
+#include "stdlib/stats/base/dmskrange.h"
+```
+
+#### stdlib_strided_dmskrange( N, \*X, strideX, Mask, strideMask )
+
+Computes the [range][range] of a double-precision floating-point strided array `x` according to a `mask`.
+
+```c
+#include <stdint.h>
+
+const double x[] = { 1.0, -2.0, 2.0 };
+const uint_8 mask = { 0, 1, 0 };
+
+double v = stdlib_strided_dmskrange( 3, x, 1, mask, 1 );
+// returns 1.0
+```
+
+The function accepts the following arguments:
+
+-   **N**: `[in] CBLAS_INT` number of indexed elements.
+-   **X**: `[in] double*` input array.
+-   **strideX**: `[in] CBLAS_INT` stride length for `X`.
+-   **Mask**: `[in] uint_8*`. If a `mask` array element is `0`, the corresponding element in `x` is considered valid and included in computation. If a `mask` array element is `1`, the corresponding element in `x` is considered invalid/missing and excluded from computation.
```

Comment:
```suggestion
-   **Mask**: `[in] uint_8*` mask array. If a `mask` array element is `0`, the corresponding element in `x` is considered valid and included in computation. If a `mask` array element is `1`, the corresponding element in `x` is considered invalid/missing and excluded from computation.
```

Reply:
Always include an initial lowercase description before the more extended description, as done in L61.